GenumKit/Parsers/StringsFileParser.swift

```diff
@@ -82,6 +125,11 @@ public final class StringsFileParser {
       self.init(key: key, translation: translation, types: types)
     }
 
+    public init(key: String, translation: String) {
+      let types = PlaceholderType.fromFormatString(translation)
+      self.init(key: key, translation: translation, types: types)
+    }
+
     private static let lineRegEx: NSRegularExpression = {
```

Comment:
This won't probably be needed anymore now that we process the file as a dictionary PLIST instead of a textual file.